Angular InstantSearch/geo-search/package.json

```diff
@@ -1,54 +1,44 @@
 {
   "name": "geo-search",
   "version": "1.0.0",
-  "private": true,
   "scripts": {
     "ng": "ng",
     "start": "ng serve --port 3000",
-    "build": "ng build --prod --progress=false",
-    "test": "ng test",
-    "lint": "ng lint",
-    "e2e": "ng e2e"
+    "build": "ng build",
+    "watch": "ng build --watch --configuration development",
+    "test": "ng test"
   },
+  "private": true,
   "dependencies": {
-    "@agm/core": "1.0.0-beta.6",
-    "@angular/animations": "6.1.10",
-    "@angular/common": "6.1.10",
-    "@angular/compiler": "6.1.10",
-    "@angular/core": "6.1.10",
-    "@angular/forms": "6.1.10",
-    "@angular/http": "6.1.10",
-    "@angular/platform-browser": "6.1.10",
-    "@angular/platform-browser-dynamic": "6.1.10",
-    "@angular/router": "6.1.10",
-    "algoliasearch": "3.33.0",
-    "angular-instantsearch": "3.0.0-beta.2",
-    "core-js": "2.6.7",
-    "instantsearch.js": "3.7.0",
-    "rxjs": "6.3.3",
-    "zone.js": "0.9.1"
+    "@agm/core": "^3.0.0-beta.0",
+    "@angular/animations": "~12.2.0",
```

Comment:
do we want to keep these as ~?

Reply:
I'm not strongly opinionated on this, but the advantage I see is that it generated less updates via dependabot. 
Or simple install, the .lock prevents it to break so no problem there. The risk is really when we add/remove things from the package.json and invalidate the .lock.
But since theses `~` come from Angular CLI itself, I think we can trust them and benefit from less dependabot PRs.